<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailDialog</class>
 <widget class="QMainWindow" name="DetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detail</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="formatedName">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>%NAME%</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="callsign">
           <property name="font">
            <font>
             <pointsize>32</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>%CALLSIGN%</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="address">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <fontweight>Medium</fontweight>
              </font>
             </property>
             <property name="text">
              <string>%ADDRESS%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cityStateZip">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <fontweight>Medium</fontweight>
              </font>
             </property>
             <property name="text">
              <string>%CITY_STATE_ZIP%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="country">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <fontweight>Medium</fontweight>
              </font>
             </property>
             <property name="text">
              <string>%COUNTRY%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWebEngineView" name="imageBrowser" native="true">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="tabPosition">
          <enum>QTabWidget::TabPosition::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::TabShape::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="detailsTab">
          <attribute name="title">
           <string>Details</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="DetailTableView" name="detailTable">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderHighlightSections">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderHighlightSections">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="bioTab">
          <attribute name="title">
           <string>Bio</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QWebEngineView" name="bioBrowser" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mapTab">
          <attribute name="title">
           <string>Map</string>
          </attribute>
          <layout class="QVBoxLayout" name="mapTabLayout">
           <item>
            <widget class="QQuickWidget" name="map" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrint"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>:/images/print.png</normaloff>:/images/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/images/floppy_disk.png</normaloff>:/images/floppy_disk.png</iconset>
   </property>
   <property name="text">
    <string>Save PDF</string>
   </property>
   <property name="toolTip">
    <string>Save details as a PDF file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DetailTableView</class>
   <extends>QTableView</extends>
   <header>detailtableview.h</header>
  </customwidget>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
  </customwidget>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>qquickwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
